plugin/completion.py

```diff
@@ -203,48 +188,61 @@ def format_completion(self, item: dict) -> sublime.CompletionItem:
         if item.get("deprecated", False):
             kind = (kind[0], '⚠', "⚠ {} - Deprecated".format(kind[2]))
 
-        text_edit = item.get("textEdit")
-        if text_edit:
-            r = text_edit["range"]
-            start, end = r["start"], r["end"]
-            # From the LSP spec: the range of the edit must be a single line range and it must contain the position at
-            # which completion has been requested.
-            row, start_col_utf16, end_col_utf16 = start["line"], start["character"], end["character"]
-            # The TextEdit from the language server might apply to an old version of the buffer. The user may have
-            # entered more characters in the meantime.
-            global _changes_since_sent
-            for change in _changes_since_sent:
-                # Thus, adjust the range for each buffer change.
-                start_col_utf16, end_col_utf16 = transform_region(start_col_utf16, end_col_utf16, change)
-            # We don't have to use region_to_range. That function clamps the unreliable input, but having unreliable
-            # (row, col) points for completions is pretty much disastrous. So we'll assume the (row, col) points are
-            # correct. Blame the language server if the regions are incorrect.
-            convert = self.view.text_point_utf16
-            item["native_region"] = (convert(row, start_col_utf16), convert(row, end_col_utf16))
-
         lsp_label = item["label"]
         lsp_filter_text = item.get("filterText")
         lsp_detail = item.get("detail") or ""
+        lsp_detail = html.escape(lsp_detail.replace('\n', ' '))
+
+        if can_resolve_completion_items or "documentation" in item:
+            doc_link = '<a href="subl:lsp_resolve_docs">Docs...</a>'
```

Comment:
Maybe say `Documentation...`? A bit more formal.

Reply:
Turned it into `Documentation`. I hope it doesn't take up too much space.